src/main/features/orders/draft/ordersDraft.ts

```diff
@@ -0,0 +1,15 @@
+import { DraftDocument } from '@hmcts/cmc-draft-store-middleware'
+import { DisagreeReason } from 'orders/form/models/disagreeReason'
+
+export class OrdersDraft extends DraftDocument {
```

Comment:
new Draft for just 1 property? I can see that it makes sense logically to keep it as separate draft but is it necessary?

Reply:
Where else would I save it? Or do you mean to not save the information at all? if the claim-store errors then it will be lost and someone could have put a lot of effort into writing their reason that would then be lost.

Reply:
my understanding is orders come after DQ, so do you think we can just use existing DQ draft? just asking! i could be wrong.

Reply:
This story is for after the order has been drawn. it's after both parties have submitted their DQ and the legal advisor has drawn the order that this flow is made available.